personal/privacyidea/class_mfaAccount.inc

```diff
@@ -828,7 +828,7 @@ class mfaAccount extends plugin
             return $this->attrs["uid"][0];
         }
 
-        return "";
+        return $this->ui->uid;
```

Comment:
Maybe add an if-clause to check if $this->ui->uid exists / isset?

Reply:
It always exists, the `uid` property is initialized in the constructor. the `class userinfo` constructor queries the uid in LDAP and populates it with the first on returned, can it ever be null/empty or wouldn't the array access on the returned attributes in the userinfo constructor already cause an exception?